modules/prebidServerBidAdapter/index.js

```diff
@@ -483,8 +484,23 @@ const OPEN_RTB_PROTOCOL = {
       tmax: _s2sConfig.timeout,
       imp: imps,
       test: getConfig('debug') ? 1 : 0,
+      ext: {
+        prebid: {
+          targeting: {
+            // includewinners is always true for openrtb
+            includewinners: true,
```

Comment:
pardon my ignorance here, but are either of these flags a breaking change?

Reply:
I do not think so,

I believe they just instruct PBS what to do with the `ext.prebid.targeting.adServerTargeting` response object.

includewinners: true ==> `hb_{}` keys are passed back

includebidderkeys: true ==> `hb_{}_{bidder}` keys are passed back

We set the latter to false to keep the number of KVPs down.
@bretg Can confirm maybe?

Reply:
thanks, that makes sense.